src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -75,12 +79,19 @@ public long amount() {
         throw new UnsupportedOperationException("amount() not yet implemented");
     }
 
-    // @todo #15:30min Implement prefix() by parsing the string representation
-    //  of transaction according to the pattern, described in the white
-    //  paper. Replace relevant test case with actual tests.
     @Override
     public String prefix() {
-        throw new UnsupportedOperationException("prefix() not yet implemented");
+        return new UncheckedText(
+            new ListOf<>(
+                new SplitText(
+                    new TextOf(
+                        this.transaction
+                    ),
+                    new TextOf(";")
+                )
+            //@checkstyle MagicNumberCheck (1 line)
+            ).get(3)
```

Comment:
@paulodamaso this will throw `ArrayIndexOutOfBoundsException` if element at index 3 is missing. Let's wrap it (with a useful message) with something like `IllegalArgumentException`, or perhaps extend it and create our own type (e.g. `TransactionFormatException`). See my other comment about validation.